OpenRA.Mods.Cnc/Traits/TDGunboat.cs

```diff
@@ -32,6 +32,7 @@ public class TDGunboatInfo : TraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		public override object Create(ActorInitializer init) { return new TDGunboat(init, this); }
 
 		public WAngle GetInitialFacing() { return InitialFacing; }
+		public Color GetTargetLineColor() { return Color.Green; }
```

Comment:
We should extract this into a `TargetLineColor` field for consistency.

Reply:
I'd rather not expose a parameter that would never be actually used.

Reply:
Ok.